docs/standard/microservices-architecture/net-core-net-framework-containers/index.md

```diff
@@ -3,11 +3,11 @@ title: Choosing Between .NET Core and .NET Framework for Docker Containers
 description: .NET Microservices Architecture for Containerized .NET Applications | Choosing Between .NET Core and .NET Framework for Docker Containers
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 05/26/2017
+ms.date: 09/11/2018
 ---
 # Choosing Between .NET Core and .NET Framework for Docker Containers
 
-There are two supported implementations for building server-side containerized Docker applications with .NET: [.NET Framework](https://www.microsoft.com/net/download/framework) and [.NET Core](https://www.microsoft.com/net/download/core). They share many .NET Standard components, and you can share code across the two. However, there are fundamental differences between them, and which implementation you use will depend on what you want to accomplish. This section provides guidance on when to choose each implementation.
+There are two supported frameworks for building server-side containerized Docker applications with .NET: [.NET Framework and .NET Core](https://www.microsoft.com/net/download/windows). They share many .NET platform components, and you can share code across the two. However, there are fundamental differences between them, and which framework you use will depend on what you want to accomplish. This section provides guidance on when to choose each framework.
```

Comment:
Drop "windows" from the download link. That way, it automatically resolves to the platform OS from the reader's browser.

Reply:
Fixed!